src/components/layout/Header/HeaderContacts/HeaderContacts.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * HeaderContacts component.
+ * @module components/ItaliaTheme/Header/HeaderContacts/HeaderContacts
+ */
+
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { UniversalLink } from '@plone/volto/components';
+import { Icon } from 'io-sanita-theme/components';
+import { Container, Row, Col } from 'design-react-kit';
+import './headerContacts.scss';
+
+const HeaderContacts = () => {
+  const items = useSelector(
+    (state) =>
+      state?.content?.data?.['@components']?.['iosanita-settings']
+        .contatti_testata,
+  );
+
+  return (
+    items && (
+      <div className="header-contacts">
+        <Container>
+          <Row>
+            {items.map((item, index) => {
+              return (
+                <Col className="contact-wrapper" key={item['@id']}>
+                  {item.description && (
+                    <span className="item-description">{item.description}</span>
+                  )}
+                  {item.tag && <span className="item-type">{item.tag}</span>}
+                  {item.link_value && (
+                    <div
```

Comment:
qui devi mostrare del testo 'slate' avendo cambiato il widget

Reply:
La gestione qui non sembra cambiare, data arriva con gli stessi parametri, sempre un HTML <p><a>, quindi ho mantenuto il dangerouslySetInnerHTML

Reply:
no, data ti arriva come html perchè hai il valore salvato in precedenza
cancella il contenuto del campo e fai una prova da 0